### PR TITLE
Removed retry, adjusted logging

### PR DESCRIPTION
**Pull request by tamas-molnar**

### CODE REVIEW

**Comment by scott-ace-newton** [comment]
:+1:

**Comment by BerniVarga** [comment]
+1

File: main.go [hunk 1]
```diff
@@ -199,7 +200,7 @@ func (httpConf httpConfigurations) readMessage(msg queueConsumer.Message) {
 	err := sendToWriter(ingestionType, strings.NewReader(msg.Body), uuid, httpConf.baseURLSlice)
 
 	if err != nil {
-		log.Errorf("Error processing msg: %v with error %v", msg, err)
+		log.Errorf("%v", err)
```
Comment (catalin-ilea) [inline main.go]: Should we log the uuid separately so that we can easily extract it and republish?

Comment (tamas-molnar) [inline main.go]: @catalin-ilea yes, done

**Pull request merged**